### PR TITLE
feat: makes the list of metadata files configurable

### PR DESCRIPTION
**Pull request by regseb**
- Add CLI flag to configure metadata files (_package.json_, _package-lock.json_, ...): `--pkg-files` and `--lock-files`.
- Add properties in configuration file to configure metadata files: `pkgFiles` and `lockFiles`.

### CODE REVIEW

**Comment by bcoe** [comment]
@jbottigliero I think you were working on a similar patch for this?

**Comment by jbottigliero** [comment]
Hey @bcoe, @regseb – yes, this feature is included in #370. 

In that PR, I've opted to keep the configuration names we're already using, but it should provide the same functionality. 

@regseb do you have any thoughts on the implementation proposed in #370?

File: test.js [hunk 1]
```diff
@@ -822,7 +823,7 @@ describe('standard-version', function () {
       shell.exec('git tag -a v1.0.0 -m "my awesome first release"')
       commit('feat: new feature!')
 
-      require('./index')({ silent: true })
+      require('./index')(Object.assign({}, defaults, { silent: true }))
```
Comment (jbottigliero) [inline test.js]: I believe manually importing and merging the defaults here is going to lead to false-positives in the tests.

**Comment by regseb** [comment]
You mean the PR https://github.com/conventional-changelog/standard-version/pull/372? Your proposal includes my modifications with additional features. So you can close my PR.

I put two comments in the other PR: https://github.com/conventional-changelog/standard-version/pull/372#pullrequestreview-272082098

**Comment by jbottigliero** [comment]
Whoops, yes #372! Thanks!

**Pull request closed without merging**